__tests__/utility.service.test.js

```diff
@@ -19,18 +20,51 @@ describe("UtilityService", () => {
   });
 
   describe("updateSearchResults", () => {
-    it("should call provided functions", () => {
+    it("should call provided functions and return expected objects", () => {
+      const mockCurrentLocation = {
+        latitude: 45.497237,
+        longitude: -73.578913,
+      };
+
+      const mockCurrentMarkerLocation = {
+        id: CURRENT_LOCATION_DISPLAY_TEXT,
+        displayName: CURRENT_LOCATION_DISPLAY_TEXT,
+        location: mockCurrentLocation,
+        searchName: CURRENT_LOCATION_DISPLAY_TEXT,
+        extraInformation: "",
+      };
+
+      let mockSetSearchResults = jest.fn();
+      let mockSetDisplayValue = jest.fn();
+
+      utilityService.updateSearchResults(
+        "test input",
+        mockSetSearchResults,
+        mockSetDisplayValue,
+        mockCurrentLocation
+      );
+
+      expect(mockSetSearchResults).toHaveBeenCalledTimes(1);
+      expect(mockSetDisplayValue).toHaveBeenCalledTimes(1);
+      expect(mockSetSearchResults.mock.calls[0][0][0]).toEqual(
+        mockCurrentMarkerLocation
+      );
+    });
+
+    it("Search results and display value should be called once and the mocked value of search results should be empty", () => {
```

Comment:
Non-blocking: nice description, however can this be reworded to match the`it` expectation? Something like `it("should show xxx so that yyy")`